components/pre_trial_pricing_modal.tsx/content.tsx

```diff
@@ -0,0 +1,296 @@
+                        {formatMessage({id: 'pretrial_pricing_modal.title', defaultMessage: 'Select a plan'})}
+                    </h1>
+                    <div>{'Choose a plan to get started'}</div>
+                </div>
+                <button
+                    id='closeIcon'
+                    className='icon icon-close'
+                    aria-label='Close'
+                    title='Close'
+                    onClick={() => {}}
+                />
+            </Modal.Header>
+            <Modal.Body className='PreTrialPricingModal__body'>
+                <div className='self-hosted-svg-left'>
+                    <LadySvg/>
+                </div>
+                <div className='self-hosted-svg-right'>
+                    <ManSvg/>
+                </div>
+                <Card
+                    topColor='#9DA7B8'
+                    plan='Starter'
+                    price={formatMessage({id: 'pretrial_pricing_modal.price.free', defaultMessage: 'Free'})}
+                    briefing={{
+                        title: formatMessage({id: 'pretrial_pricing_modal.briefing.starter', defaultMessage: 'Starter comes with...'}),
+                        items: [
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.storage', defaultMessage: '{storage}GB file storage'}, {storage: '50'}),
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.starter.messageHistory', defaultMessage: 'Limited message history'}),
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.starter.boardCards', defaultMessage: '{boards} Boards cards per server'}, {boards: '500'}),
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.starter.teamCount', defaultMessage: 'Limited to one team'})],
+                    }}
+                    buttonDetails={{
+                        action: () => {},
+                        text: formatMessage({id: 'pretrial_pricing_modal.btn.upgrade', defaultMessage: 'Upgrade'}),
+                        disabled: isStarter,
+                        customClass: ButtonCustomiserClasses.grayed,
+                    }}
+                    planDisclaimer={formatMessage({id: 'pretrial_pricing_modal.planDisclaimer.starter', defaultMessage: 'This plan has data restrictions.'})}
+                    planLabel={
+                        isStarter ? (
+                            <PlanLabel
+                                text={formatMessage({id: 'pretrial_pricing_modal.planLabel.currentPlan', defaultMessage: 'CURRENT PLAN'})}
+                                color='#3DB887'
+                                bgColor='#FFFFFF'
+                                firstSvg={<CheckMarkSvg/>}
+                            />) : undefined}
+                />
+                <Card
+                    topColor='#4A69AC'
+                    plan='Professional'
+                    price='$10'
```

Comment:
For this we should probably look at the actual price in the cloud products in redux and avoid hardcoding the value